mlir/lib/Bindings/Python/PybindUtils.h

```diff
@@ -14,6 +14,7 @@
 #include "llvm/ADT/Twine.h"
 #include "llvm/Support/DataTypes.h"
 
+#include <pybind11/numpy.h>
```

Comment:
unused i think?

Reply:
It is used in `IRAttributes.cpp` now to use the numpy bitpacking functions. I didn't include it directly in `IRAttributes.cpp`, because it seems like all `pybind`` includes have been collected in `PybindUtils.h`

Reply:
nah definitely (as a rule) put `include`s in cpp (ie implementation) unless it's absolutely necessary to put them in the headers. Even then, we often "pre declare" instead of including the header.